actions/image.search.by.expression.yaml

```
@@ -0,0 +1,140 @@
+---
+description: Search for images with a selected expression
+enabled: true
+entry_point: src/openstack_actions.py
+name: image.search.by.expression
+parameters:
+  timeout:
+    default: 5400
+  lib_entry_point:
+    default: workflows.search_by_expression.search_by_expression
+    immutable: true
+    type: string
+  query_type:
+    default: ImageQuery
+    immutable: true
+    type: string
+  as_admin:
+    default: True
+    immutable: true
+    type: boolean
+  cloud_account:
+    description: The clouds.yaml account to use whilst performing this action
+    required: true
+    type: string
+    default: "dev"
+    enum:
+      - "dev"
+      - "prod"
+  output_type:
+    default: "to_string"
+    type: string
+    enum:
+      - "to_html"
+      - "to_objects"
+      - "to_props"
+      - "to_string"
+    description: "
+    A string representing how to return the results of the query
+      - 'to_html' - a tabulate table (in html)
+      - 'to_props' - properties dicts as a python list
+      - 'to_objects - as a list of openstack resources
+      - 'to_string' - a tabulate table"
+    required: true
+  property_to_search_by:
+    default: "image_id"
```

This default won't work as it isn't in the enum, should be changed to something else, e.g. image_creation_progress
```suggestion
    default: "image_creation_progress"
```

---

Have changed